mule-shaders-module/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.mule.extensions</groupId>
+        <artifactId>mule-core-modules-parent</artifactId>
+        <version>1.1.9</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-shaders-module</artifactId>
+    <version>1.0.0-rc1</version>
+    <packaging>jar</packaging>
+    <name>Shader Utilities Module</name>
+
+    <properties>
+        <mule.extensions.shade.plugin.version>1.0.4</mule.extensions.shade.plugin.version>
+        <mulesoftLicenseVersion>1.4.0</mulesoftLicenseVersion>
+        <license.maven.plugin.version>4.2</license.maven.plugin.version>
+        <licensePath>LICENSE_HEADER_CPAL.txt</licensePath>
+        <licenseYear>2023</licenseYear>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.mule.runtime.plugins</groupId>
+            <artifactId>mule-extensions-shade-plugin</artifactId>
+            <version>${mule.extensions.shade.plugin.version}</version>
+            <scope>provided</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.mycila</groupId>
```

Comment:
The plugin is also declared for json module, in order to customise the license configuration.

We shouldn't change this without spending time figuring out why the license headers needed to be customised for json module in the first place, but for parity reasons, I think it's more important to keep it working the same way and just update the license year.

I was hoping to avoid bundling too many changes unrelated to reflective access, because it makes the PR more noisy and less likely for reviewers to want to get involved.